zap/src/parser/convert.rs

```diff
@@ -88,6 +88,7 @@ impl<'src> Converter<'src> {
 		let (write_checks, ..) = self.boolean_opt("write_checks", true, &config.opts);
 		let (typescript, ..) = self.boolean_opt("typescript", false, &config.opts);
 		let (manual_event_loop, ..) = self.boolean_opt("manual_event_loop", false, &config.opts);
+		let (typescript_max_tuple_length, ..) = self.num_opt("typescript_max_tuple_length", 11.0, &config.opts);
```

Comment:
Shouldn't this be `10.0` as that's what you specified the deafult was on the docs.